### DISCUSSION TITLE
how to disable all extensions ?

### DISCUSSION THREAD

**Discussion by 10100010010001001010** [axmolengine/axmol#1652] [Q&A] [answered]
I saw a topic here about this, but now I can’t find it either in discussions or in issues. Tell me how to properly create a project without extensions?

**Comment by rh101** [comment, marked as answer]
The wiki has all the info you need.  https://github.com/axmolengine/axmol/wiki/FAQ

**Reply by 10100010010001001010** [reply]
thank you
